clouddriver-openstack/src/main/groovy/com/netflix/spinnaker/clouddriver/openstack/model/OpenstackLoadBalancer.groovy

```diff
@@ -30,30 +30,35 @@ import org.openstack4j.model.network.ext.LbPool
 @Canonical
 @JsonIgnoreProperties(['nameRegex','namePattern','descriptionRegex','descriptionPattern'])
 class OpenstackLoadBalancer implements LoadBalancer, Serializable, LoadBalancerResolver {
-    String type = OpenstackCloudProvider.ID
-    String account
-    String region
-    String id
-    String name
-    String description
-    String status
-    String protocol
-    String method
-    String ip
-    Integer externalPort
-    String subnet
-    Set<PoolHealthMonitor> healthChecks
-    Set<LoadBalancerServerGroup> serverGroups = Sets.newConcurrentHashSet()
 
-  static OpenstackLoadBalancer from(LbPool pool, OpenstackVip vip, OpenstackSubnet subnet, OpenstackFloatingIP ip,
+  String type = OpenstackCloudProvider.ID
+  String account
+  String region
+  String id
+  String name
+  String description
+  String status
+  String protocol
+  String method
+  String ip
+  Integer externalPort
+  String subnetId
+  String subnetName
+  String networkId
+  String networkName
+  Set<PoolHealthMonitor> healthChecks
+  Set<LoadBalancerServerGroup> serverGroups = Sets.newConcurrentHashSet()
+
```

Comment:
I have been using builders instead of static methods ... Feels like it gives more flexibility for each use case.